services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -11,6 +11,7 @@ const {
 } = require('../../../databases/models');
 const Getstream = require('../../../vendor/getstream');
 const RedisBlockHelper = require('../../redis/helper/RedisBlockHelper');
+const BetterSocialScoreBlockUser = require('../score/block-user');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `BetterSocialScoreBlockUser` has been added. This is a new dependency for this file, so please ensure that the module is available in your project and it exports the function `BetterSocialScoreBlockUser`. If not, it will cause a runtime error.
```diff
+ const BetterSocialScoreBlockUser = require('../score/block-user');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->